src/20231102_Multi_AaveFundingUpdates/AaveFundingUpdates.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Aave Funding Updates"
+author: "TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-aave-funding-update/15194"
+---
+
+## Simple Summary
+
+This AIP aims to consolidate Aave DAO's stablecoin holdings. It is divided into different parts that are detailed below. Part one is depositing DAI into aEthDAI, part two is migrating aDAI and aUSDT into aEthDAI and aEthUSDT, part three bridges assets from Polygon to Mainnet and part four swaps some holdings into GHO.
+
+## Motivation
+
+This proposal intends to ensure the DAO is well capitalised. A separate DAO budget publication will be shared during Q4 2023.
+
+The DAO has sufficient USDC, DAI and USDT deposited across the various instances of Aave Protocol to support existing streams and the AURA purchase 3.
+
+However, with the emergence of new service provider proposal expected, the DAO needs to adjust its stable coin holdings. This publication supports transferring additional funds from Polygon to Mainnet and acquiring GHO on the market to ensure the DAO is well capitalised for the next 6 months. It also aims to move assets from the old V2 to the newer V3 aToken.
+
+## Specification
+
+- Deposit 1M units of DAI from Treasury into Aave v3
+- Migrate all aDAI from Aave v2 to v3
+
+- Redeem and transfer 1.7M USDC to Ethereum Treasury
+- Redeem and transfer 0.5M DAI to Ethereum Treasury
+- Redeem and transfer 0.75M USDT to Ethereum Treasury
+
+Swap the following asset holdings to GHO:
```

Comment:
ARC USDC missed here

Reply:
hey @rustboyar is there a tool to do the ARC transfer and withdrawal?
the owner of the funds to Transfer is GovernanceV3, while to withdraw from ARC, it's the ARC timelock.
I think I have to do it in 2 steps, a payload to transfer, and then a payload to withdraw, so was hoping maybe there's a way to do both?

Reply:
removing ARC 'cause it's just a painful process